test/numa/README.md

```diff
@@ -0,0 +1,156 @@
+# CRI Resource Manager - NUMA node tests
+
+## Prerequisites
+
+Install:
+- `docker`
+- `govm`
```

Comment:
I would add here that once govm is installed tests might fail with an error like this:
```
Creating /home/kli/vms/imagestime="2020-08-13T19:49:03+03:00" level=info msg="Pulling govm/govm:latest image"
time="2020-08-13T19:49:03+03:00" level=fatal msg="Error when creating the new VM: repository name must be canonical"
```

This is a known issue and there is an [open bug](https://github.com/govm-project/govm/issues/22) about this.
Here is an easy workaround:
```
git clone https://github.com/govm/govm
docker build govm -f govm/Dockerfile -t govm/govm:latest
```

Reply:
Yeah, I'll add it.